gdc_client/upload/parser.py

```diff
@@ -9,15 +9,20 @@
 from . import exceptions
 
 from .client import GDCUploadClient
+from .. import log
 
 
+logger = log.get_logger('upload-client')
+
 def validate_args(parser, args):
     """ Validate argparse namespace.
     """
-    if args.manifest or args.identifier:
+    if args.manifest or args.file_ids:
+        if args.identifier:
```

Comment:
Oh I think I see what you think and I believe you are confused about what `identifier` is. It's basically just a flag that gets set to true if the user specifies it. It's not a UUID. The DTT will exit with an error if `manifest` and `file_ids` are both missing even if `identifier` is specified. But I guess we can also warn the user about it's use of the `identifier` before exiting